problem2.py

```diff
@@ -1 +1,6 @@
-
+print("\nWelcome to the popeyes chicken distributor.")
+total_chicken = int(input("\nHow much chicken do you need to distribute?: "))
+total_students = int(input("\nHow many students do you need to distribute it amongst?: "))
+chicken_per_student = (total_chicken // total_students)
+remainder = total_chicken - (chicken_per_student*total_students)
+print("\nEach student receives " + str(chicken_per_student) + " chicken(s) and Mr. Fabroa will receive " + str(remainder) + " chicken(s).")
```

Comment:
use whitespacing to break up your program into more readable logical chunks, and use comments to describe the chunks
```
print("\nWelcome to the popeyes chicken distributor.")

# get chicken and student numbers
total_chicken = int(input("\nHow much chicken do you need to distribute?: "))
total_students = int(input("\nHow many students do you need to distribute it amongst?: "))

# compute chicken per student and teacher
chicken_per_student = (total_chicken // total_students)
remainder = total_chicken - (chicken_per_student*total_students)

# show results
print("\nEach student receives " + str(chicken_per_student) + " chicken(s) and Mr. Fabroa will receive " + str(remainder) + " chicken(s).") 
```